### PR TITLE
cloudstack: exoscale: fix boilerplate

### PR DESCRIPTION
**Pull request by resmo**
##### SUMMARY
* fix future-import-boilerplate
* fix metaclass-boilerplate

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
cloudstack
exoscale

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by resmo** [comment]
ready_for_review

**Review by felixfontein — APPROVED** [review]

**Review by Andersson007 — APPROVED** [review]
LGTM
shipit

**Comment by Andersson007** [comment]
Some ideas for the future:
1. add ```seealso:``` section with links to related modules and official documentation
2. add ```elements:``` to params of list type if they are presented anywhere

File: lib/ansible/modules/cloud/cloudstack/cs_affinitygroup.py [hunk 1]
```diff
@@ -200,7 +202,6 @@ def main():
     argument_spec = cs_argument_spec()
     argument_spec.update(dict(
         name=dict(required=True),
-        affinty_type=dict(removed_in_version='2.9'),
```
Comment (felixfontein) [inline lib/ansible/modules/cloud/cloudstack/cs_affinitygroup.py:203]: I would prefer to move that into a separate PR. Feature removal should also have a changelog (`removed_features:`) and potentially also a porting guide entry (no idea where though).

**Review by dpassante — APPROVED** [review]
shipit

File: lib/ansible/modules/cloud/cloudstack/cs_affinitygroup.py (hunk 1 above)
Comment (resmo) [reply lib/ansible/modules/cloud/cloudstack/cs_affinitygroup.py:203]: this deprecation was due to a typo in a param name and is not noteworthy becuase it should be clear to use the one without a typo...

**Comment by resmo** [comment]
Thanks all for the reviews!
